dashboards-observability/public/components/event_analytics/explorer/log_patterns/patterns_table.tsx

```diff
@@ -15,7 +15,7 @@ import { PatternTableData } from 'common/types/explorer';
 import { reduce, round } from 'lodash';
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { SELECTED_PATTERN_REGEX } from '../../../../../common/constants/explorer';
+import { FILTERED_PATTERN } from '../../../../../common/constants/explorer';
```

Comment:
this works too, can you remove the regex?

Reply:
there are more unused variables, will clean up in a separate PR